.github/workflows/commit-message-validation.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       - name: Validate commit messages
         run: |
           # Regex for Conventional Commits specification
-          COMMIT_REGEX="^(build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test)(\([^\)]*\))?:\s?(EXPOSED-[0-9]+\s?)?.+$"
+          COMMIT_REGEX="^(build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test)(!)?(\([^\)]*\))?:\s?(EXPOSED-[0-9]+\s?)?.+$"
```

Comment:
Modified regular expression to allow for optional "!" (to denote breaking changes) after the commit type